config/co2eq_parameters.json

```diff
@@ -713,784 +789,2209 @@
     }
   },
   "fallbackZoneMixes": {
+    "defaults": {
```

Comment:
I put this as 'defaults' because this is exactly what we do above for the emissions Factors 
https://github.com/tmrowco/electricitymap-contrib/blob/78bebbf0161ef6fe086df2b04cc0f781b341b155/config/co2eq_parameters.json#L2-L5

Reply:
if we change one, I guess we should change both?

Reply:
good catch! We should indeed keep both consistent.
Another way to fix the inconsistency is to add `zoneOverrides` as is done for the emissionFactors.
The important part is to not have the "defaults" key and a zone key at the same level, because our system might mistakenly use "defaults" as a zone key - which it isn't.
I hope it makes sense!

Reply:
makes sense. changed